examples/README.md

```diff
@@ -1,13 +1,13 @@
-The csp-billing-adapter requires a configuration file, (defaults to `~/.config/csp_billing_adapter/config.yaml`)
-in order to run successfully. The configuration files in in the `examples` directory 
+The csp-billing-adapter requires a configuration file, (defaults to `/etc/csp_billing_adapter/config.yaml`)
```

Comment:
Technically, since the default path cannot be changed without changing the code right now (no CLI options to do so, and not env var setting that can be overridden) this isn't so much a default path as a hardcoded path... Just saying.

Reply:
Should we add a config option to do this or have an env variable check for now?

Reply:
i added an env variable check to adapter.py to allow an override